### ISSUE TITLE
Fix Unit Administrator not seeing hidden resources

### ISSUE THREAD

**Issue by petterill** [City-of-Helsinki/respa#462]

**Comment by suutari-ai** [comment]
Yep, I can confirm that the Respa backend was coded to hide the hidden resources from the unit managers as there was no specification of it in the original documentation and hiding them was safer choice when nobody explicitly requested them to be shown.  The issue should be easy to solve though: Update the filters in https://github.com/City-of-Helsinki/respa/blob/master/resources/models/resource.py#L136. Remember to update the docs too at https://github.com/City-of-Helsinki/respa/blob/master/docs/permissions.rst

**Comment by petterill** [comment]
@otiala Addition: so this should apply to Unit Group Administrator, Unit Administrator and Unit Manager.
